sky/adaptors/aws.py

```diff
@@ -173,11 +173,9 @@ def client(service_name: str, **kwargs):
 
 def botocore_exceptions():
     """AWS botocore exception."""
-    from botocore import exceptions
-    return exceptions
+    return botocore.exceptions
```

Comment:
Q: Are these changes due to previous `from botocore import exceptions` failing?

Reply:
It won't fail. This is just to simplify the implementation as our `LazyImport` will raise the error directly. : )